docs/diagrams/AddSkillCommandSequenceDiagram.puml

```diff
@@ -13,10 +13,10 @@ box Model MODEL_COLOR_T1
 participant ":Model" as Model MODEL_COLOR
 end box
 
-[-> LogicManager : execute("addskill 1 s/C")
+[-> LogicManager : execute("addskill 1 s/Java")
```

Comment:
I've also just realized this after reading your PR, but the diagram input and the text description input (directly above it) do not match for most of the commands. Do you think it would be good to standardize them? (Change either the code block in description or diagram). If so, could you do it as well in this PR?